addons/settings/script_component.hpp

```diff
@@ -58,8 +58,10 @@
 #define MENU_OFFSET_COLOR 1.0
 #define MENU_OFFSET_COLOR_NEG -0.7
 
+#define IS_ADMIN serverCommandAvailable "#shutdown"
```

Comment:
maybe a put this macro in "script_macros_common.hpp"?
could be useful elsewhere